server/src/lib/model/db.ts

```diff
@@ -29,13 +29,12 @@ export default class DB {
 
   /**
    * Normalize email address as input.
-   * TODO: add validation here.
    */
-  private formatEmail(email?: string): string {
-    if (!email) {
-      return '';
+   private formatEmail(email?: string): string {
+	let re = new RegExp('/^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/');
```

Comment:
please check the indentation here. Also, would it make sense to use another variable name to describe what the regex does? maybe something like invalidRegex?

Reply:
(as discussed: as the RegEx tests for a valid email, the if below should be inverted ;))